docs/managing_deis/backing_up_data.rst

```diff
@@ -195,6 +195,12 @@ Note that you'll need to specify the SSH port when using Vagrant:
 Finishing up
 ~~~~~~~~~~~~
 
+.. note::
+
+    As of version 1.11.0, the following steps are no longer necessary because
+    the ``deis-controller`` component will handle them automatically upon
+    starting.
```

Comment:
Should we delete this whole "Finishing up" topic if it's not needed? Then the docs are consistent with the codebase, and there is no need to mention the next unreleased version, which may confuse people looking at the default "latest" documentation.